fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -1273,7 +1273,9 @@ private void startNonLeaderDaemonThreads() {
             metastoreEventsProcessor.start();
         }
 
-        connectorTableMetadataProcessor.start();
+        if (Config.enable_background_refresh_connector_metadata) {
```

Comment:
@DorianZheng Please double check. You are responsible for this configuration once online